.github/workflows/pre-compile_llvm.yml

```diff
@@ -35,18 +32,19 @@ jobs:
         run: |
           sudo add-apt-repository ppa:ubuntu-toolchain-r/ppa
```

Comment:
I think the logic would be easier to understand if you add the following here, and delete `env:` section above:
```
echo "cpp=g++" >> $GITHUB_ENV
```

Reply:
fixed